q2_gneiss/plot/_plot.py

```diff
@@ -134,8 +134,28 @@ def f(x):
             sample_palette = pd.Series(
                 sns.color_palette("Set2", len(c.value_counts())),
                 index=c.value_counts().index)
-            balance_boxplot(balance_name, data, y=c.name, ax=ax,
-                            palette=sample_palette)
+            try:
+                balance_boxplot(balance_name, data, y=c.name, ax=ax,
+                                palette=sample_palette)
+            except TypeError as e:
```

Comment:
Instead of performing this check in a catch block, why not just preemptively perform this check, before we run the `balance_subplot` command?

Reply:
I did it this way on the off chance that at some point in the future it does start to accept numerical data types, do you think I should change it around?

Reply:
Yes, the preemptive approach limits our coupling to matplotlib/gneiss/numpy, which I think is ideal.